packages/wallet/.env

```diff
@@ -4,16 +4,18 @@ DEFAULT_GAS_PRICE=20000000000
 ENABLE_SOLC_OPTIMIZER=TRUE
 FIREBASE_PROJECT =  'rock-paper-scissors-dev'
 FIREBASE_API_KEY =  'AIzaSyAlGe17xjJjfoJ_KDYjCREg7ZL4ns61Chc'
+# TODO: Currently using existing channels fails so this is disabled
+# We need to address this
 # Controls if channels and related info are stored/loaded from IndexedDB
-USE_STORAGE=TRUE
+USE_STORAGE=FALSE
```

Comment:
Why is this an environment variable? Shouldn't this be in the webpack config? Are there other contexts where this would ever be set differently?

Reply:
Since the `wallet` always attempts to use an existing channel to fund it's pretty handy to be able to stop this behaviour and force the `wallet` to start from a blank slate when you're looking into issues around funding with new channels.

I don't think this should be in webpack config, since our webpack config should be a static configuration, and `.env` should be used for toggles that a developer may way to change when running the `wallet`.

Longer term this will probably go as the `wallet` gets more sophisticated with storage.

Reply:
I see. So, this is only needed for `yarn start` / a hot-reloading dev server config.